public/docs/_examples/user-input/dart/lib/todo_list.dart

```diff
@@ -0,0 +1,41 @@
+library user_input.todo_list;
+
+import 'dart:html';
+
+import 'package:angular2/angular2.dart';
+
+@Component(selector: 'todo-list')
+@View(
```

Comment:
Ok good to know. I've seen both ways through out other examples so i wasn't sure if we wanted to unify on one way over the other for all examples, or if we wanted to show different ways in different parts.